packages/vendure-plugin-drop-off-points/src/drop-off-points.plugin.ts

```
@@ -0,0 +1,51 @@
+    const carrierNames = options.carriers.map((p) => p.name);
+    const duplicateCarrierNames = carrierNames.filter(
+      (name, index) => carrierNames.indexOf(name) !== index
+    );
+    if (duplicateCarrierNames.length) {
+      throw new Error(
+        `[${loggerCtx}] Carrier name should be unique, but found the following names more than once: ${duplicateCarrierNames.join(
+          ','
+        )}`
+      );
+    }
+    this.options = options;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)